index.html

```diff
@@ -49,10 +49,86 @@ <h2>LET'S CONNECT</h2>
               <a href="https://github.com/isaac-ej"><img src="img/Vector (1).svg" alt=""></a>
             </li>
           </ul>
-        
         </section>
-   </header>
+     </header>
+       <main class="card-container">
+         <!-- Work section -->
+         <section class="container">
+          <div class="bg">
+            <div class="snap">
+              <img src="/img/Snapshoot Portfolio.png">
+            </div>
+            <div class="left-block">
+              <div class="primary-text">
+                <h5 class="tonic">TONIC </h5>
+                <div class="frame">
+                  <p>Canopy</p>
+                  <ul>
+                    <li><div class="spans"></div></li>
+                    <li>Backend Dev</li>
+                    <li><p class="spans"></p></li>
+                    <li>2015</li>
+                  </ul>
+                </div>
+              </div>
+            </div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+            <div class="techs">
+              <ul>
+                <li>HTML</li>
+                <li> CSS</li>
+                <li>Javascript</li>
+              </ul>
+            </div>
+            <button>See Projects</button>
+          </div>
+         </section>
```

Comment:
- [ ] Great progress on your work so far! 👍🏿 However, I've noticed that you have only created one card out of the four cards provided in the Figma template. According to the project requirements, I recommend that you create all the cards present in the Figma design for this particular project. It's important to ensure that all the necessary components are implemented. Keep up the good work! ✌🏿

Reply:
For your information, we were required to create only two sections:
work section and about section it was to choose only one